index.d.ts

```diff
@@ -0,0 +1,22 @@
+import { EventEmitter } from "events";
+import { BaseMQTTSNPacket, MQTTPacket } from "./packet";
```

Comment:
Shouldn't it be `MQTTSNPacket` instead of `MQTTPacket` ?
```suggestion
import { BaseMQTTSNPacket, MQTTSNPacket } from "./packet";
```